src/index.js

```diff
@@ -73,41 +73,22 @@ export default function boolean(subject, clipping, operation) {
   //console.timeEnd('subdivide edges');
 
   // console.time('connect vertices');
-  const result = connectEdges(sortedEvents, operation);
+  const contours = connectEdges(sortedEvents, operation);
   //console.timeEnd('connect vertices');
 
-  const out = [];
-  for (var i = 0; i < result.length; i++) {
-    const contour = result[i];
-    const points = contour.points;
-    if (contour.external) {
-      if (points[0][0] !== points[points.length - 1][0] ||
-          points[0][1] !== points[points.length - 1][1]
-      ) {
-        points.push([points[0][0], points[0][1]]);
+  // Convert contours to polygons
+  const polygons = [];
+  for (let contour of contours) {
```

Comment:
This threw errors for me running `npm run build`
````
(plugin buble) CompileError: Transforming for-of statements is not fully supported. Use `transforms: { forOf: false }` to skip transformation and disable this error, or `transforms: { dangerousForOf: true }` if you know what you're doing. (88:6)
````
It was simple enough to swap out with regular for loops which I think would be more normal.

Reply:
And similar in other places in `index.js` using `for (let foo of bar) {`

Reply:
Good point, I've fixed the problem with `npm run build` now.